### PR TITLE
Pass query params along to the redirected route.

### PR DESCRIPTION
**Pull request by clhamilton**
This naively adds the query params to the redirected route.
I've included tests for it, but can certainly add to them if you want, just let me know.

### CODE REVIEW

File: tests/acceptance/query-params-test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+
+var App;
+
+module('Query params will be passed along - Integration', {
+  setup: function() {
+    App        = startApp();
```
Comment (thoov) [inline tests/acceptance/query-params-test.js]: Could you fix the indentation here

**Comment by thoov** [comment]
Thanks @clhamilton. I left one comment and after you fix that I will merge

**Comment by clhamilton** [comment]
Thanks @thoov, the indentation has been fixed.

**Comment by joeyespo** [comment]
Any chance this PR will be released to npm soon? Thanks!

**Comment by thoov** [comment]
@joeyespo I just pushed 4.1.0 which has this change

**Pull request merged**